pkg/auth/migrate/rbac_to_acl.go

```diff
@@ -107,14 +109,55 @@ func RBACToACL(ctx context.Context, svc auth.Service, doUpdate bool, creationTim
 			if errors.Is(err, auth.ErrAlreadyExists) {
 				warnings = multierror.Append(warnings, err)
 			} else if err != nil {
-				return err
+				return nil, err
 			}
 		}
 
 		messageFunc(group.DisplayName, *newACL, warnings)
 	}
-
-	return nil
+	var usersWithPolicies []string
+	hasMoreUser := true
+	afterUser := ""
+	for hasMoreUser {
+		// get membership groups to user
+		users, userPaginator, err := svc.ListUsers(ctx, &model.PaginationParams{
+			After:  afterUser,
+			Amount: pageSize,
+		})
+		if err != nil {
+			return nil, err
+		}
+		for _, user := range users {
+			// get policies attracted to user
+			hasMoreGroupPolicy := true
+			afterGroupPolicy := ""
+			for hasMoreGroupPolicy {
+				userPolicies, userPoliciesPaginator, err := svc.ListUserPolicies(ctx, user.Username, &model.PaginationParams{
+					After:  afterGroupPolicy,
+					Amount: pageSize,
+				})
+				if err != nil {
+					return nil, fmt.Errorf("list user policies: %w", err)
+				}
+				if len(userPolicies) > 0 {
+					usersWithPolicies = append(usersWithPolicies, user.Username)
+				}
+				if !doUpdate {
+					break
+				}
```

Comment:
if you break here, you basically lose the next page if the user has more policies, no?

Reply:
Yes but I'm fine with that because I'm not in update mode and I only want to warn that this user will have policies-detached